### ISSUE TITLE
Lock tabs?

### ISSUE THREAD

**Issue by dubcanada** [bwinton/TabCenter#959]
Hello,

I'd like a way too lock the tabs, I have it set so it just shows the favicon like
![screen shot 2017-02-24 at 4 33 51 pm](https://cloud.githubusercontent.com/assets/120325/23319950/18c39830-faaf-11e6-96c3-30b00248b482.png)

and it works fine, but every now and then I accidentally click the resizer and resize it then I want and I have too readjust it.

Any thoughts?

**Comment by Keith94** [comment]
Good idea. A temporary solution (e.g. for [Stylish](https://addons.mozilla.org/firefox/addon/stylish/)) is using this code:

```css
#verticaltabs-splitter {
    pointer-events: none;
}
```

**Comment by ericawright** [comment]
Any ideas on how to  add the ability to unlock it again, once the user decides they want to resize it? We'd need a button or something to indicate the user wants it locked and to indicate they want it unlocked. I don't see this being implemented in an easy-to-use way.

**Comment by Keith94** [comment]
@ericawright You could add a tab context menu option for locking/unlocking.

**Comment by bwinton** [comment]
Man, _no-one_ discovers context-menus…  🙂

**Comment by Keith94** [comment]
Well, Tree Style Tab has "fix position and width of tab bar" option.

![firefox_2017-02-27_07-07-15](https://cloud.githubusercontent.com/assets/5490615/23366409/740ab742-fcbb-11e6-88da-1e5beb4f7daf.png)

**Comment by bwinton** [comment]
Yes, Tree Style Tab has a lot of features aimed at more advanced users.  We're trying to build something a little simpler with Tab Center.

**Comment by SoftVision-CiprianMuresan** [comment]
@bwinton, what if we add a checkbox in about:addons->Extensions->Tab Center, much like the Reverse Tab Order one?

**Comment by bwinton** [comment]
That seems like a long way to go to get to that functionality…  (Ideally there would be something near the control itself.)  I wonder if we could solve it by having an explicit grabby control that shows up when you hover over the tabs, and have that be the only way to resize?  Then there would be an obvious place to resize it, but it would be much smaller than all along the border…
